test/test.js

```diff
@@ -1547,10 +1549,12 @@ test('simulate invalid alter _lightMyRequest.isDone without end', (t) => {
     replied = true
   })
 
-  setTimeout(() => {
-    t.equal(end, false)
-    t.equal(replied, false)
-  }, 10)
+  process.nextTick(() => {
+    setImmediate(() => {
+      t.equal(end, false)
+      t.equal(replied, false)
+    })
+  })
```

Comment:
Can you just use one of those?

Reply:
Sadly, I cannot. Using either one of them will fail the test, using both of them still faster than ```setTimeout(fn, 10)``` for 20ms

Reply:
```process.nextTick``` and ```setImmediate``` result vary on machine. I didnt expect it will fail the test (it is success on my computer). I will revert the new commit.